.gitignore

```diff
@@ -1,9 +1,352 @@
-﻿bin/
-obj/
+﻿## Ignore Visual Studio temporary files, build results, and
```

Comment:
This change seems unnecessary extensive.

Reply:
I've noticed commited .vs or .vscode directories previously so IMO it's good to just add a standard VS .gitignore, so it will be less likely for contributors to commit some unnecessary files

Reply:
Is it the complete default?

Reply:
Yes, check out https://github.com/github/gitignore/blob/master/VisualStudio.gitignore